config/manager/kustomization.yaml

```diff
@@ -2,3 +2,8 @@ resources:
 - manager.yaml
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+patches:
+# Protect the /metrics endpoint by putting it behind auth.
+# If you want your controller-manager to expose the /metrics
+# endpoint w/o any authn/z, please comment the following line.
+- path: manager_auth_proxy_patch.yaml
```

Comment:
I am not sure about this change. There is a reason why the generate code decided to put it in the top most overlay and not in the manager's overlay

Reply:
i have  seen repo's moving this as per their requirements/customizations, and I don't see any problem in moving this to the manager as well, If you see it as a problem let me know what problem it can cause and alternative suggestion as well

Reply:
@Madhu-1 I would say, let's not move it from it original location if there is no real reason to do so

Reply:
added a comment on top of the files as per discussion